backend/src/main/java/de/neuefische/backend/service/DoctorService.java

```diff
@@ -15,11 +18,13 @@
 
     private final DoctorRepo doctorRepo;
     private final UtilService utilService;
+    private final IdService idService;
 
     @Autowired
-    public DoctorService(DoctorRepo doctorRepo, UtilService utilService) {
+    public DoctorService(DoctorRepo doctorRepo, UtilService utilService, IdService idService) {
```

Comment:
```suggestion
    public DoctorService(DoctorRepo doctorRepo,  IdService idService) {
```
see my comment in UtliService